### DISCUSSION TITLE
Linux Repository

### DISCUSSION THREAD

**Discussion by OishiiCha** [AntonyCorbett/OnlyT#436] [Ideas]
Hi, is there a possibility of having a Linux Repository for OnlyT or a Deb file? 
I'm not sure how much would be involved in doing so or if it is even possible. Just would be easier to run OnlyT on a lighter machine such as an SBC. I think ARM Architecture would be preferable, however I imagine that that would require emulation with WINE. Currently OnlyT doesn't work with WINE.

**Comment by CharllysFernandes** [comment]
It doesn't have a repository Linux or .deb file. But I think it should work fine using applications like Winehq or Winapps.

see more: 

https://github.com/Fmstrat/winapps
or
https://www.winehq.org/

**Reply by Fuseteam** [reply]
it does work with winehq.org unfortunately as mentioned by the OP
winapps is a windows virtual machine.......that sounds overkill for a simple timer

**Comment by Fuseteam** [comment]
there seems there is someone who made a port for linux: https://github.com/AntonyCorbett/OnlyT/discussions/566#discussioncomment-15181744
